src/test/client/nonblocking.cc

```diff
@@ -312,4 +318,93 @@ TEST_F(TestClient, LlreadvLlwritevZeroBytes) {
 
   client->ll_release(fh);
   ASSERT_EQ(0, client->ll_unlink(root, filename, myperm));
-}
\ No newline at end of file
+}
+
+TEST_F(TestClient, LlreadvLlwritevSignalInterrupt) {
+  /*Simulate an interrupted system call by sending a SIGALRM signal to the
```

Comment:
I don't think this needs to be a part of the functional test. Such tests are better off to be written as unit tests.

Reply:
okay, i'll remove that test case

Reply:
removed